src/home/profile/Profile.js

```diff
@@ -33,56 +35,76 @@ ScreenProps<> & InjectedProps
     const { profile } = profileStore;
     return (
       <>
-        <View style={styles.container}>
-          <NavHeaderWithButton title="Profile" buttonFn={this.settings} buttonIcon="settings" />
-          <View style={styles.header}>
-            <View style={styles.title}>
-              <Text type="header2" style={styles.name}>{profile.name}</Text>
-            </View>
+      <NavHeaderWithButton title="Profile" buttonFn={this.settings} buttonIcon="settings" />
+      <Container gutter={1} style={styles.container}>
+        <Content scrollEnabled={false}>
+          <View style={{borderBottomColor: 'lightgray', borderBottomWidth: 1, marginBottom: 12}}>
             <Avatar
               size={avatarSize}
               style={styles.avatar}
               {...profile.picture}
             />
           </View>
-        </View>
+          <View>
+            <View style={styles.informationContainer}>
+              <Text style={styles.header}>Name</Text>
+              <Text style={styles.information}>{profile.name}</Text>
```

Comment:
We can use Firebase to get the name of the current user instead of using ProfileStore